src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceService.kt

```diff
@@ -103,14 +103,12 @@ class LicenceService(
     standardConditionRepository.saveAllAndFlush(entityStandardLicenceConditions + entityStandardPssConditions)
 
     auditEventRepository.saveAndFlush(
-      transform(
```

Comment:
Most of the changes here are just removing the pattern of creating a model entity and immediately transforming it into an entity object. 

Now we just create the entity directly